scheduler/pkg/agent/model_state.go

```diff
@@ -153,7 +152,7 @@ type modelVersion struct {
 }
 
 func (version *modelVersion) getVersionMemory() uint64 {
-	return version.versionInfo.GetModel().GetModelSpec().GetMemoryBytes()
+	return version.versionInfo.GetModel().GetModelSpec().GetMemoryBytes() * uint64(version.versionInfo.ModelConfig.GetInstanceCount())
```

Comment:
The fact that atm we'll be using the instance count like this (simply ignoring the resource) is another reason why I've proposed we remove the resource field for now.